jujugui/static/gui/src/app/components/user-profile/user-profile.js

```diff
@@ -555,9 +559,10 @@ YUI.add('user-profile', function() {
     _generateBundleRow: function(bundle) {
       var id = bundle.id;
       var services = [];
-      var serviceNames = Object.keys(bundle.services);
+      var applications = bundle.applications || bundle.services || {};
+      var serviceNames = Object.keys(applications);
       serviceNames.forEach((serviceName, idx) => {
-        var service = bundle.services[serviceName];
+        var service = applications[serviceName];
```

Comment:
I cannot find the consideration above. but the code falls back to .services so Juju 1 should be ok.